odc/geo/cog/_tifffile.py

```diff
@@ -573,6 +583,33 @@ def _gdal_level(compression: str, default=None) -> Any:
     return (predictor, compression, compressionargs)
 
 
+def _gdal_sample_description(sample: int, description: str) -> str:
+    """Make XML line of GDAL metadata.
+
+    :param band: Sample / band number in data array.
+    :param description: Band name in data array.
+
+    :return: GDAL XML metadata line to place in TIFF file.
+    """
+    # GDAL does double escaping; see frmts/gtiff/geotiff.cpp.
+    # We also double escape to maximize compatibility with tools expecting GDAL-generated metadata.
+    double_escaped_description = xml_escape(xml_escape(description))
+    return f'<Item name="DESCRIPTION" sample="{sample}" role="description">{double_escaped_description}</Item>'
+
+
+def _gdal_sample_descriptions(xx: xr.DataArray) -> List[str]:
+    """Translate ``long_name`` attribute (if present) to GDAL sample description metadata.
+
+    :param xx: Pixels as :py:class:`xarray.DataArray` backed by Dask
```

Comment:
"backed by Dask", is not a requirement here

Reply:
Thanks, done.